modules/migrating-to-multi-arch-cli.adoc

```diff
@@ -0,0 +1,68 @@
+// Module included in the following assemblies:
+//
+// * updating/migrating-to-multi-payload.adoc
+
+:_content-type: PROCEDURE
+[id="migrating-to-multi-arch-cli_{context}"]
+= Migrating to a cluster with multi-architecture compute machines using the CLI
+
+.Prerequisites
+
+* You must be logged in as a user with `cluster-admin` privileges on all clusters.
+* Your {product-title} version must be up to date to at least version 4.13.0. 
++
+For more information on how to update your cluster version, see _Updating a cluster using the web console_ or _Updating a cluster using the CLI_ listed in the Additional resources section.
+* You installed the OpenShift CLI (`oc`).
+* Your oc client must be updated to at least verion 4.13.0. To verify that your oc client version will support the `--to-multi-arch` command, run the following command:
++
+[source,terminal]
+----
+$ oc adm upgrade --help
+----
+* You must verify `RetrievedUpdates=True` in Cluster Version Operator (CVO) conditions by running the following command:
++
+[source,terminal]
+----
+$ oc adm upgrade
+----
++ 
+For more information about cluster version condition types, refer to _Understanding cluster version condition types_ in the Additional resources section.
+* Your {product-title} cluster must be installed on either the AWS or Azure platform. 
++
+For more information on selecting a supported platform for your cluster installation, see _Selecting a cluster installation type_ in the Additional resources section.
+
+
+.Procedure
+. Set the channel to `stable-<4.y>` or `fast-<4.y>` with the following command: 
++
+[source,terminal]
+----
+$ oc adm upgrade channel <channel>
+----
++
+For more information about channels, refer to _Understanding update channels and releases_ listed in the Additional resources section.
+
+. Migrate to the mult-architecture payload with following command:
++
+[source,terminal]
+----
+$ oc adm upgrade --to-multi-arch.
+----
+
+[NOTE]
+====
+The `.` is important to include at the end of the `oc adm upgrade --to-multi-arch` command.
+====
```

Comment:
This looks odd. AFAIK there are no such requirement.

Reply:
@shellyyang1989 mentioned that the period was necessary at the end of the command. Is that not the case?